### PR TITLE
D3 graph component

### PR DESCRIPTION
**Pull request by taras**
Created `{{d3-graph}}` component that can be used as a root for a d3 graph and updated tests and readme to use this component.

### CODE REVIEW

**Comment by ivanvanderbyl** [comment]
I like where this is going.

**Comment by taras** [comment]
@ivanvanderbyl sweet :)

File: README.md [hunk 1]
```diff
@@ -44,6 +45,32 @@ You can view a demo of a few ways to use these helpers [here](http://locusenergy
 
 ## Usage
 
+### `{{d3-graph graph}}`
+
+`d3-graph` is the only component in this addon. It's used to provide root level selection where your graph will be rendered.
```
Comment (zigahertz) [inline README.md]: What do you think about rewording this sentence - to something like:
"It's used to provide root level selection to render discrete D3 elements, such as SVG `<svg>` and groups `<g>`."

Comment (taras) [inline README.md]: Ok, I'll make his change.

**Pull request merged**